storage.conf

```diff
@@ -69,6 +69,9 @@ additionalimagestores = [
 # and vfs drivers.
 #ignore_chown_errors = "false"
 
+# Inodes is used to set a maximum inodes of the container image.
```

Comment:
Can we mention the format here as well?

Reply:
Format is actually not supported for Inodes, I removed the line from the man page.